mmdet/datasets/pipelines/transforms.py

```diff
@@ -2022,6 +2026,9 @@ def __call__(self, results):
             dict: Result dict with mosaic transformed.
         """
 
+        if random.uniform(0, 1) > self.prob:
```

Comment:
Please add unit test to prob.

Reply:
> Please add unit test to prob.

I add an assertion test for invalid `prob` in `tests/test_data/test_pipelines/test_transform/test_transform.py`.
Does the unit test need to set different probabilities like `test_img_augment.py`?
https://github.com/open-mmlab/mmdetection/blob/e045607843f439f064791506257e9b4fdbf268c0/tests/test_data/test_pipelines/test_transform/test_img_augment.py#L138-L142

Reply:
Yes